### DISCUSSION TITLE
Deleting a project/context not possible?

### DISCUSSION THREAD

**Discussion by Grydpype** [ransome1/sleek#485] [General]
Apparently I am no longer able to delete a project; when opening the options for a project (or a context), I can rename it & store the changes, but the "delete" button is always greyed out. Of course the context/project disappears automatically once all the tasks are gone, but I would nevertheless be able to delete them manually.

What am I doing wrong?

Cheers, 
Oliver.

**Comment by ransome1** [comment]
<img width="320" alt="Bildschirmfoto 2023-05-15 um 21 52 49" src="https://github.com/ransome1/sleek/assets/11188741/044c4992-4797-4c29-9583-1074225dddc3">

So it doesn't look like this on your side? Can you share some information on your OS, sleek version and maybe a screenshot?

**Reply by Grydpype** [reply]
Good morning,
I thought I had closed this item, because in the end it was a misinterpretation on my part.

The "Delete" button is coloured a faint red on my screen, which I interpreted as "greyed out". When I used it nonetheless at some point just out of curiosity, it worked perfectly.

So the only remaining question for me is: why does the label's colour differ from the "save" button's colour? I am unaccustomed to fonts using different colours on buttons; but now that I found this out, there's no need to change anything.

Thanks again for the beautiful software; I use it every day.

Regards,
Oliver.
